src/components/household/InviteConfirm.js

```diff
@@ -12,14 +12,13 @@ import { GoogleLogin } from 'react-google-login';
 const InviteConfirm = (props) => {
   useEffect(() => {
     axiosWithAuth()
-      .post('/members/household/accept-invite', { hash })
+      .post('/members/household/accept-invite', { hash, permissionLevel })
```

Comment:
This is related to the backend requested change but it doesn't stop anything by having it, so we can always come back later to clean it a bit

Reply:
Done